src/main/java/com/homework/steps/LoginSteps.java

```diff
@@ -46,7 +46,7 @@ public void correct_pasword() {
         loginPage.providePassword();
     }
 
-    @Then("^I am loged in$")
+    @Then("^I am logged in$")
     public void i_am_loged_in() {
         assertThat("URL contains inbox", inboxPage.urlContainsText("inbox"), is(Boolean.TRUE));
```

Comment:
Hmm. Hamcres assertions are more human readable but maybe junit assert.true would be better